core/Application/Kernel/PluginList.php

```diff
@@ -42,7 +42,8 @@ class PluginList
         'ExampleReport',
         'MobileAppMeasurable',
         'Provider',
-        'TagManager'
+        'TagManager',
+        'CustomDirPlugin'
     );
```

Comment:
@tsteur If you're trying to get this to load in the test, adding it to extraPluginsToLoad in Fixture should work.

Reply:
@diosmosis I'm trying to fix this test: https://travis-ci.org/matomo-org/matomo/jobs/489810408#L1048

as in UI test I did `testEnvironment.pluginsToLoad = ['CustomDirPlugin'];`... so hoping this way it will be maybe updated or work?

Reply:
@diosmosis didn't fix the error. do you have any idea how to make sure it updates the plugin? I wouldn't even really have expected an update.

Reply:
That might do it, they should get added in TestingEnvironmentManipulator.php

Reply:
The customdirplugin has a visit dimension, I guess that triggers the update. I think the problem is that UITestFixture uses the OmniFixture dump, but that of course wouldn't load CustomDirPlugin. UITestFixture.php calls Fixture::updateDatabase(), but I guess that isn't working here...

If you set UITestFixture::$extraPluginsToLoad to `['CustomDirPlugin']`, does that work?

Reply:
Oh I think I see the problem now. The plugin has to be loaded before the fixture is setup, but setup-fixture command is made to be the first thing that is done in the test (see screenshot-testing/support/app.js).